packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1479,7 +1479,14 @@ function emitOperationGroups(
     string,
     OperationGroup
   >();
-  for (const operationGroup of listOperationGroups(context, client)) {
+  const clientOperations: HrlcOperation[] = [];
+  for (const operation of listOperationsInOperationGroup(context, client)) {
+    clientOperations.push(emitOperation(context, operation, "", rlcModels));
+  }
+  if (clientOperations.length > 0) {
+    addHierarchyOperationGroup(clientOperations, groupMapping);
+  }
+  for (const operationGroup of listOperationGroups(context, client, true)) {
```

Comment:
it seems that we will prompt the all operation groups into top-level for nested operation groups.

Not sure this is finalized decision?